index.bs

```diff
@@ -22,6 +22,8 @@ Boilerplate: omit conformance, omit feedback-header
 spec:fetch; type:dfn; for:/; text:request
 spec:webidl; type:dfn; text:resolve
 spec:infra; type:dfn; text:user agent
+spec:infra; type:dfn; for:/; text:list
+spec:permissions; type:dfn; for:/; text: permission task source
```

Comment:
This should not be necessary. I guess the issue is that the permissions spec is not exporting this definition? If so, then maybe we shouldn't use it, instead of using Bikeshed features to bypass that protection.

I'd suggest filing an issue/PR on the permissions spec to discuss exporting this task source.

Reply:
Filed https://github.com/w3c/permissions/issues/206

Reply:
Done